### PR TITLE
Update getting started docs

### PR DESCRIPTION
**Pull request by maciaszczykm**
- [x] Update getting started docs for v2.13
- [x] Update getting started docs for master

### CODE REVIEW

File: content/kubermatic/master/getting_started/manage_projects/_index.en.md [hunk 1]
```diff
@@ -11,35 +11,46 @@ Kubermatic organizes your clusters into projects. Projects allow you to share cl
...
+![Add member dialog](/img/master/getting_started/manage_projects/projects-member.png)
 
 * **Editors** have write access to your project and can manage clusters, nodes and SSH keys.
 * **Viewers** have read-only access and can only view the existing resources.
 
 You can change the role for a user or remove them altogether at any time.
 
 After adding a user to a project, the project will immediately show up in the user's project list.
+
+### Step 5 - Manage Project Service Accounts
```
Comment (zreigz) [inline content/kubermatic/master/getting_started/manage_projects/_index.en.md:47]: +1

**Comment by kgroschoff** [comment]
I like it very much 👍 

btw in case you don't know it, if you open the `deploy/netlify` details, you can see a live preview 😃

**Pull request merged**